### ISSUE TITLE
Link title optimieren

### ISSUE THREAD

**Issue by lizardat** [oveleon/contao-glossary-bundle#56]
Im Javascript wird der Title des Links aus dem Begriff erstellt, also aus "xxx" wird ein <a title="xxx" ...>xxx</a>

Ein nettes Feature wäre, wenn man für jedes Glossar einen Zusatztext definieren könnte, der entsprechend den Title ergänzt... z.B. "xxx - Begriff im Glossar nachschlagen".

Begründung: Im Sinne der Barrierefreiheit sollte der Sinn eines Links verständlich sein (https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html). Eine simple Wiederholung des Text im Title kann man sich ersparen.

Danke.

**Comment by zoglo** [comment]
There might be more important stuff to enhance accessibility and meet the standards than having a descriptive title but that's something out of scope for the next 4 months.

In this case, the title could even be omited completely, wdyt?

> The purpose of each link can be determined from the link text alone or from the link text
https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html

Adding the title is not a criterion:
https://www.w3.org/WAI/WCAG21/Techniques/html/H33

**Comment by lizardat** [comment]
yes, if link text and title are the same, title should not be included.

and: providing a link purpose is a very important criteria for accessibility. That's why its already on level A of WCAG. While visually a difference to "regular" links can be provided for the glossary links (via CSS), this should also be available for non-visual users.

**Comment by zoglo** [comment]
Are we only talking about the automatic link-markup or also about the title itself that would be clickable?

**Comment by lizardat** [comment]
I don't use hovercards, only the automatic keyword conversion. Didn't have a look at hovercards so far.
For now I just extended javascript with 
`... case"a":o.title=e+' - im Glossar nachschlagen' ...`
Not safe for updates and only in only language... ;)

**Comment by zoglo** [comment]
You should be able to add multi language support via config as seen here
https://github.com/contao/contao-demo/pull/53/files

then provide translations via contao and override your glossary_config JS template and put them in via `<?= $this->trans() ?>`

**Comment by zoglo** [comment]
Feel free to create a PR for it if you have time

**Comment by zoglo** [comment]
@lizardat just wanted to follow up if you are gonna create a PR or not since you already had a working solution🙃?

**Comment by zoglo** [comment]
@lizardat I will close this feature request of yours since there wasn't any activity of yours and you didn't react after being mentioned multiple times.

If you want us to do it for you, you are free to comission us financially for any features you need in the future.
